gradle/libs.versions.toml

```
@@ -0,0 +1,135 @@
+[versions]
+# KotlinX
+coroutines = "1.5.1"
+
+# Core
+koin = "3.1.2"
+androidx-core = "1.6.0"
+androidx-appcompat = "1.3.1"
+androidx-activity = "1.3.0"
+androidx-fragment = "1.3.6"
+androidDesugarLibs = "1.1.5"
+
+# Lifecycle extensions
+androidx-lifecycleExtensions = "2.3.1"
+
+# UI
+androidx-constraintLayout = "2.1.0"
+google-material = "1.4.0"
+androidx-webkit = "1.4.0"
+modernAndroidPreferences = "2.1.0"
+
+# Network
+jellyfinSdk = "1.0.1"
+jellyfinSdkLocal = "latest-SNAPSHOT"
+jellyfinSdkSnapshot = "master-SNAPSHOT"
+jellyfinSdkSnapshotUnstable = "openapi-unstable-SNAPSHOT"
+okhttp = "4.9.1"
+coil = "1.3.1"
+
+# Media
+androidx-media = "1.4.0"
+androidx-mediarouter = "1.2.4"
+exoPlayer = "2.14.2"
+playServices = "20.0.0"
+
+# Room
+androidx-room = "2.3.0"
+
+# Monitoring
+timber = "4.7.1"
+leakCanary = "2.7"
+redScreenOfDeath = "0.1.3"
+
+# Testing
+junit = "5.7.2"
+kotest = "4.6.1"
+mockk = "1.12.0"
+androidx-test-runner = "1.4.0"
+androidx-test-espresso = "3.4.0"
+
+[libraries]
+# KotlinX
+coroutines-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version.ref = "coroutines" }
+coroutines-android = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-android", version.ref = "coroutines" }
+
+# Core
+koin = { group = "io.insert-koin", name = "koin-android", version.ref = "koin" }
+androidx-core = { module = "androidx.core:core-ktx", version.ref = "androidx-core" }
+androidx-appCompat = { module = "androidx.appcompat:appcompat", version.ref = "androidx-appcompat" }
```

```suggestion
androidx-appcompat = { module = "androidx.appcompat:appcompat", version.ref = "androidx-appcompat" }
```

Officially called "Appcompat" without uppercase C

---

Right, that's a copy paste error. I fixed it for some other deps.